src/lib/libwebaudio.js

```diff
@@ -73,7 +73,12 @@ var LibraryWebAudio = {
 
   // Call this function from JavaScript to get the Web Audio object corresponding to the given
   // Wasm handle ID.
-  $emscriptenGetAudioObject: (objectHandle) => emAudio[objectHandle],
+  $emscriptenGetAudioObject: (objectHandle) => {
+#if ASSERTIONS || WEBAUDIO_DEBUG
+    emAudioExpectNodeOrContext(objectHandle, 'emscriptenGetAudioObject');
+#endif
```

Comment:
The above call to `emAudioExpectNodeOrContext()` seems restrictive.

It prevents users from testing if a given handle has been deleted? E.g. one would not be able to write code like this anymore:

```
if (emscriptenGetAudioObject(someHandle)) {
  console.log('someHandle exists, accessing it...');
} else {
  console.log('someHandle has been deleted, doing something else (maybe deinitializing some JS side structure at shutdown?)');
}
```

Reply:
Indeed, that is the idea.    None of our use cases for the API seems to use it in this way, and none of the other APIs in the this file allow invalid handles to be passed.   Unless there is specific use case for allowing invalid handles here I suggest we be consistent.  (At least until someone asks for this ability).